yt_dlp/extractor/common.py

```diff
@@ -379,6 +379,8 @@ class InfoExtractor:
                     'private', 'premium_only', 'subscriber_only', 'needs_auth',
                     'unlisted' or 'public'. Use 'InfoExtractor._availability'
                     to set it
+    media_type:     The type of media, for instance a full show, an excerpt, a highlight, a trailer
+                    or something similar that classifies what kind of video or audio this is.
```

Comment:
`media_type` will also need to be documented in the readme

Reply:
I've just added a commit, where I've shortened the description to match the others